package.json

```diff
@@ -46,5 +46,10 @@
   "lint-staged": {
     "*": "prettier --write --cache --ignore-unknown",
     "*.{ts,js,tsx,jsx}": "eslint --cache --fix"
+  },
+  "dependencies": {
+    "@types/bingmaps": "^0.0.1",
```

Comment:
These types should live in the packages/pages-components/package.json instead of the root level one. This repo is set up as a monorepo (multi package repo).